packages/poi/lib/handle-options.js

```diff
@@ -9,6 +9,14 @@ const { inferHTML, readPkg } = require('./utils')
 module.exports = co.wrap(function * (options) {
   const loadExternalConfig = new LoadExternalConfig({ cwd: options.cwd })
 
+  if (options.component) {
+    console.log('> Bundling component')
+    options.format = 'cjs'
```

Comment:
what about supporting `umd` format as well, like:

```bash
poi --component {LibraryName}
```

When `options.component` is a string we build it in UMD format.

Reply:
Ok, sounds good